moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -132,27 +132,6 @@ bool Servo::readParameters()
   bool have_scene_collision_proximity_threshold = rosparam_shortcuts::get(
       LOGNAME, nh, "scene_collision_proximity_threshold", parameters_.scene_collision_proximity_threshold);
 
-  double collision_proximity_threshold;
-  // 'collision_proximity_threshold' parameter was removed, replaced with separate self- and scene-collision proximity
-  // thresholds
-  // TODO(JStech): remove this deprecation warning in ROS Noetic; simplify error case handling
-  if (nh_.hasParam("collision_proximity_threshold") &&
-      rosparam_shortcuts::get(LOGNAME, nh, "collision_proximity_threshold", collision_proximity_threshold))
-  {
-    ROS_WARN_NAMED(LOGNAME, "'collision_proximity_threshold' parameter is deprecated, and has been replaced by separate"
-                            "'self_collision_proximity_threshold' and 'scene_collision_proximity_threshold' "
-                            "parameters. Please update the servoing yaml file.");
-    if (!have_self_collision_proximity_threshold)
-    {
-      parameters_.self_collision_proximity_threshold = collision_proximity_threshold;
-      have_self_collision_proximity_threshold = true;
-    }
-    if (!have_scene_collision_proximity_threshold)
-    {
-      parameters_.scene_collision_proximity_threshold = collision_proximity_threshold;
-      have_scene_collision_proximity_threshold = true;
-    }
-  }
```

Comment:
Is there any harm in leaving this here for a couple of releases (it was in the last one) and removing it after the Nov 20 release?